wemake_python_styleguide/violations/complexity.py

```diff
@@ -456,3 +458,24 @@ class TooManyElifsViolation(ASTViolation):
     #: Error message shown to the user.
     error_template = 'Found too many `elif` branches'
     code = 223
+
+
+class UnusedElseViolation(ASTViolation):
+    """
+    Forbids to use ``else`` without ``break`` in ``for`` loop.
+
+    Reasoning:
+        ``else`` block in ``for`` loops will be
```

Comment:
That's a wrong reasoning. `else` blocks will always be used.

```python
>>> for x in 'ab':
...     print(x)
... else:
...     print(1)
... 
a
b
1
```

So, can you please rename this rule? I guess `RedundantForElseViolation` is better.
And please, also change the reasoning.